backend/server.js

```diff
@@ -25,5 +25,4 @@ app.use("/api/products", productRoutes);
 
 app.listen(PORT, () => {
     connectDB();
```

Comment:
Possibly add a try catch to see if connectDB will fail. Unless failsafe mechanism is already inside the connectDB() function